homeassistant/components/cert_expiry/__init__.py

```diff
@@ -49,40 +44,3 @@ async def _async_finish_startup(_):
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
     return await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
-
-
-class CertExpiryDataUpdateCoordinator(DataUpdateCoordinator[datetime | None]):
```

Comment:
This does refactoring and adds new features. Those are not allowed to be combined in a single PR.

Please also see: <https://developers.home-assistant.io/docs/review-process>